### PR TITLE
Sign here, please! Cargo may optionally collect signatures on delivered requested crates for a small bonus.

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

Crates that are ordered via requests will get a signature form on them. The orderer of the crate can sign said form to give cargo a money bonus when they ship the manifest. If the incorrect person signs the manifest, it ruins the manifest, instead turning it into a -80 credit deduction on return.

- [x] add money bonus, thinking about half a good manifest worth

Cargo is now rated roundend based on how many signatures they fill, and given achievements if they have a very high signature fill rate! It's in the same place as Service ratings, and no achievements will be given if there isn't enough orders or no signatures were signed that shift.

@ArcaneMusic economy changes, you need to be notified 🤓

## Why It's Good For The Game

This encourages cargo techies to get out there and get those dang signatures (secretly, it's actually PLAYER INTERACTION MUAHAHAHA)

But also, it's a soft push to get cargo techies to deliver quickly AND manually if they want the other person to sign. An angry recipient definitely won't 👉 

## Changelog
:cl:
add: Cargo Signatures are now an optional bonus you can seek out, if you can get someone who requested a crate to sign the manifest themselves! Better deliver quick, as you cannot forge the signature (Centcom will deduct funds for your blunder)
/:cl:

### CODE REVIEW

**Comment by Iamgoofball** [comment]
Idk2

**Review by Iamgoofball — APPROVED** [review]
Idk2

File: code/__HELPERS/roundend.dm [hunk 1]
```diff
@@ -520,60 +538,79 @@
 	log_econ("Roundend service income: [tourist_income] credits.")
 	switch(tourist_income)
 		if(0)
-			parts += "<span class='redtext'>Service did not earn any credits...</span><br>"
+			parts += "<span class='neutraltext'>service did not interact with the tourist program enough for a grade.</span><br>"
```
Comment (maxymax13) [inline code/__HELPERS/roundend.dm:541]: lowercase intentional?

File: code/datums/achievements/job_achievements.dm [hunk 2]
```diff
@@ -11,24 +11,37 @@
...
 
 /datum/award/achievement/jobs/service_good
 	name = "Centcom Grade: Exemplary Service"
-	desc = "Centcom is very impressed with your department!"
+	desc = "More than 5000 credits, wow! Centcom is very impressed with your department!"
 	database_id = MEDAL_GOOD_SERVICE
-	icon = "chem_tut"
 
-//civilian achievies! while not recognized by the code, it is recognized by our hearts
+//for cargo techies, and the quartermaster!
```
Comment (maxymax13) [inline code/datums/achievements/job_achievements.dm:31]: nothing for the hop?

File: code/__HELPERS/roundend.dm (hunk 1 above)
Comment (tralezab) [reply code/__HELPERS/roundend.dm:541]: Yes, before these announcements had switching caps and that's bad

File: code/datums/achievements/job_achievements.dm (hunk 2 above)
Comment (tralezab) [reply code/datums/achievements/job_achievements.dm:31]: The hop doesn't earn achievements for service's work either, so...

Comment (maxymax13) [reply code/datums/achievements/job_achievements.dm:31]: that's fucked up

**Comment by maxymax13** [comment]
finally even more things for cargo to do other than eat hot chip and order emitters and guns

**Comment by YakumoChen** [comment]
Soon we'll need a cargo cloak for being top tier mail boys

**Comment by maxymax13** [comment]
i wish there was an easier way to pick up mail to deliver it, maybe mail bags could have the "hoover up all the mail when you click a tile" storage function like some other things do

**Comment by tralezab** [comment]
@maxymax13 they already do that UNROBUST

status of this pr: SOMETHING is wrong with it but i forgot

File: tgui/packages/tgui/interfaces/PaperSheet.js [hunk 3]
```diff
@@ -168,7 +168,7 @@ const checkAllFields = (txt, font, color, user_name, bold=false) => {
   }
   if (replace.length > 0) {
     for (const o of replace) {
-
+      act("fieldwritten", { value: o.raw_text });
```
Comment (stylemistake) [inline tgui/packages/tgui/interfaces/PaperSheet.js:171]: Where the fuck is act defined? This is 100% an error.

**Pull request closed without merging**